### DISCUSSION TITLE
Direct Link Requires Development Access

### DISCUSSION THREAD

**Discussion by wesshaw** [Budibase/budibase#5706] [Help] [answered]
I've been using Budibase for about 6 months now writing a QR code tagging application and I've just noticed something strange.  I have generated a couple thousand QR code stickers that all resolve to ~ https://my.budibase.app.com/my_app#/tag/:uuid.  The idea is that a Basic user that has access to that page should be able to navigate directly to an info page for each tag when they scan the QR code.  The odd bit is that the user has to have the 'Development Access' flag on the User account for it to resolve correctly, otherwise the user is taken to the 'Apps' landing page ~ https://my.budibase.app.com/builder/apps.  When they click through to the correct app they are just taken to the Home page in the app and not to the URL that was originally requested.  It was resolving correctly in a past version, possibly v1.0.73 but I'm not certain.  Is there another configuration that I need to make somewhere or is this a bug?

**Comment by melohagan** [comment, marked as answer]
Hi @wesshaw 

I'm on version **v1.0.108** locally and it seems to be working fine. What version are you on? Is your route URL definitely correct? 

Maybe share some screenshots of your User Management configuration.

**Reply by wesshaw** [reply]
My mistake, I used the development URL 🤪 .  Off to reprint thousands of stickers..........
